akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -316,6 +316,8 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
   }
   var exitingConfirmed = Set.empty[UniqueAddress]
 
+  def myTeam = cluster.settings.Team
```

Comment:
Though the more I hear it "in action" I think we may need to revisit naming here...
It can be confusing with "the us team" - do we mean the cluster or the organization, sentences become somewhat ambiguous and also pointing towards the team being an active thing "the team does something", which does not really match. Perhaps best discussed in a call though hm, not necessarily now, or shall I ticketify final bikeshedding?